GOVERNANCE.md

```diff
@@ -8,10 +8,17 @@ Committee (CTC) which is responsible for high-level guidance of the project.
 The [nodejs/node](https://github.com/nodejs/node) GitHub repository is
 maintained by Collaborators who are added by the CTC on an ongoing basis.
 
+A Collaborator is defined as any individual having commit access to any
```

Comment:
This is a great clear statement, but it is in apparent contradiction with both the preceding sentence and the immediately following sentence. I'm a fan of trying to use terminology in an inclusive way, but this doc needs to be treated with considerable care.